src/Ng/Models/RegistrationIndexPackage.cs

```diff
@@ -63,7 +63,46 @@ public Uri Registration
         public RegistrationIndexPackageDetails CatalogEntry
         {
             get;
-            private set;
+            set;
+        }
+
+        /// <summary>
+        /// Indicates if the package is a NuGet package or a local (fake) package.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsLocalPackage
+        {
+            get
+            {
+                return this.Id.Scheme.Equals("file", StringComparison.OrdinalIgnoreCase);
+            }
+        }
+
+        /// <summary>
+        /// Indicates if the package is a Microsoft package.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsMicrosoftPackage
```

Comment:
Just FYI, it's unclear if this is the final logic for determining a Microsoft package. We could end up with a fixed list of packages.